typings/index.d.ts

```diff
@@ -1,25 +1,28 @@
 import * as React from 'react';
 
+type CB = () => void
+
 export interface DownshiftProps {
     children: ChildrenFunction;
-    defaultHighlightedIndex?: number;
+    defaultHighlightedIndex?: number | null;
     defaultSelectedItem?: any;
     defaultInputValue?: string;
     defaultIsOpen?: boolean;
     getA11yStatusMessage?: (options: A11StatusMessageOptions) => any;
     itemToString?: (item: any) => string;
     onChange?: (selectedItem: any, stateAndHelpers: ControllerStateAndHelpers) => void;
-    onStateChange?: (options: StateChangeOptions, stateAndHelpers: ControllerStateAndHelpers) => void;
+    onStateChange?: (changes: StateChangeOptions, stateAndHelpers: ControllerStateAndHelpers) => void;
```

Comment:
I prefer to keep this named as `options` to be consistent with everything else in here.